Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.mm

```diff
@@ -749,18 +815,18 @@ - (BOOL)prefersStatusBarHidden
     return ensureModelAndInterface(contextId).second;
 }
 
-const VideoPresentationManagerProxy::ModelInterfacePair* VideoPresentationManagerProxy::findModelAndInterface(PlaybackSessionContextIdentifier contextId) const
+std::optional<std::reference_wrapper<const VideoPresentationManagerProxy::ModelInterfacePair>> VideoPresentationManagerProxy::findModelAndInterface(PlaybackSessionContextIdentifier contextId) const
```

Comment:
Wrapping a reference in a std::optional adds no benefit to returning a pointer. It's no safer, just harder to use and IMHO less readable. this is an unnecessary change